recipe/meta.yaml

```diff
@@ -36,11 +38,17 @@ requirements:
     - packaging
 
 test:
+  source_files:
+    - fastparquet/test
```

Comment:
The _Source Distribution_ on `PyPI` doesn't contain the `test` folder, see https://pypi.org/project/fastparquet/#files